### PR TITLE
Ratu service tables to lowercase and adapting to koatuu.json

### PR DESCRIPTION
**Pull request by Anntroy**
Deleted category items in fields like ratu_city.name.

### CODE REVIEW

File: data_ocean/services/ratu_service.py [hunk 1]
```diff
@@ -32,15 +34,41 @@ class RatuConverter(Converter):
...
+    #creating dictionary & lists for registration items that had writed to db
     region_dict = {} # dictionary uses for keeping whole model class objects
     district_list = list() # lists use for keeping cells content
     city_list = list()
     citydistrict_list = list()
 
     bulk_manager = BulkCreateManager(CHUNK_SIZE)
+
+    #changing from records like: "волинська обл." to "волинська область"
+    def clear_region_name(self, region):
```
Comment (Baranovsky1986) [inline data_ocean/services/ratu_service.py:48]: One trivial comment. Why the name of this method is different from those below (clear instead of clean)?

Comment (Anntroy) [reply data_ocean/services/ratu_service.py:48]: It's a mistake. I'll change it

File: data_ocean/services/ratu_service.py [hunk 2]
```diff
@@ -32,15 +34,41 @@ class RatuConverter(Converter):
...
+        region = region.lower()
+        region = re.sub(r"обл\.", "область", region)
+        return region.strip()
+
+    #changing from records like: "донецький р-н" to "донецький район", "р.райони вінницької області" to "райони вінницької області", "райони міста Київ" to "райони м.Київ"
+    def clean_district_name(self, district):
+        district = district.lower()
+        district = re.sub(r"р-н", "район", district)
+        district = re.sub(r"р\.", "", district)
+        district = re.sub(r"райони міста", "райони м.", district)
```
Comment (IegorSoboliev) [inline data_ocean/services/ratu_service.py:58]: is that correct renaming "райони м."?

File: data_ocean/services/ratu_service.py [hunk 3]
```diff
@@ -127,14 +156,14 @@ def save_to_citydistrict_table(self, record, region, district, city):
...
-    def save_to_street_table(self, record, region, district, city, citydistrict):    
+    def save_to_street_table(self, record, region, district, city, citydistrict):
         if record['STREET_NAME']:
             street = Street(
                 region=region, 
                 district=district,
                 city=city,
                 citydistrict=citydistrict,
-                name=record['STREET_NAME']
+                name=record['STREET_NAME'].lower()
```
Comment (IegorSoboliev) [inline data_ocean/services/ratu_service.py:166]: Do we really need to have a street`s and others names lower case? We are going to show all  these values to the public))

Comment (criticus) [reply data_ocean/services/ratu_service.py:167]: it's better to store them in one style in the db for faster search. We will modify it on the frontend for UI

**Comment by criticus** [comment]
looks ok to me

**Pull request merged**